ibm_catalog.json

```diff
@@ -0,0 +1,219 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-secrets-manager-private-cert",
+        "label": "Cloud automation for Secrets Manager Private Certificate.",
```

Comment:
why it has `.`?
Name should be - `Cloud automation for Secrets Manager private certificate`